src/amocrm/services/group_pusher.py

```diff
@@ -0,0 +1,25 @@
+from amocrm.dto import AmoCRMGroup
+from amocrm.models import AmoCRMProductGroup
+from app.services import BaseService
+from products.models import Group
+
+
+class AmoCRMGroupPusher(BaseService):
```

Comment:
```suggestion
class AmoCRMGroupsPusher(BaseService):
```

Он пушит все группы, а не одну.